deployment/examples/malfunction-indicator-light/deploy-indicator-light-service.sh

```diff
@@ -0,0 +1,102 @@
+#!/bin/bash
```

Comment:
I'd prefer if the script name reflects the prefix of the YAML files it used, i.e. `deploy_mil_service.sh`-

Reply:
My thought was that it could be confusing that the overall use case shares the same name as one of its components. But looking at this again I agree that my current solution might have made it even more confusing :| because the name mil-service is already "established".  I will change the file name.